server/server_test.go

```diff
@@ -54,19 +56,14 @@ func TestPublish(t *testing.T) {
 	publisher := &publisherMock{
 		persistedIndex: 42,
 	}
-	shipper, err := NewShipperServer(publisher)
+	shipper, err := NewShipperServer(DefaultConfig(), publisher)
 	defer func() { _ = shipper.Close() }()
 	require.NoError(t, err)
 	client, stop := startServer(t, ctx, shipper)
 	defer stop()
 
 	// get the current UUID
-	pirCtx, cancel := context.WithCancel(ctx)
-	consumer, err := client.PersistedIndex(pirCtx, &messages.PersistedIndexRequest{})
-	require.NoError(t, err)
-	pir, err := consumer.Recv()
-	require.NoError(t, err)
-	cancel() // close the stream
+	pir := getPersistedIndex(t, ctx, client)
```

Comment:
What does `pir` stand for here? Persistent index r... something? Reference?

Reply:
`PeristedIndexReply`, it's the returned type.